lib/matplotlib/figure.py

```diff
@@ -2637,7 +2638,7 @@ def set_layout_engine(self, layout=None, **kwargs):
             Users and libraries can define their own layout engines and pass
             the instance directly as well.
 
-        kwargs: dict
+        kwargs : dict
```

Comment:
```suggestion
        **kwargs
```
is our standard notation. From a user perspective, `kwargs` is not a parameter name, and there is no dict involved.

Reply:
Correct! Didn't think that much, just added the space...